influxdb3_telemetry/src/stats.rs

```diff
@@ -75,6 +97,16 @@ impl<T: Default + Num + Copy + NumCast + PartialOrd> Stats<T> {
     }
 }
 
+/// Generic function to calculate min/max/avg from another set of stats.
+/// This function works for all types of numbers (unsigned/signed/floats).
+/// It calculates min/max/avg by using already calculated min/max/avg for
+/// possibly a higher resolution.
+///
+/// For eg. the minimum number of
+/// lines written in a minute interval, is compared with other minute
+/// minute intervals sampled so far. If 1st sample's minimum was 20
```

Comment:
> For eg. the minimum number of lines written in a minute interval, is compared with other minute minute intervals sampled so far.

I think this sentence needs some wordsmithing.

Reply:
@hiltontj I've merged this PR after updating the example, here is the link for the update from the other [PR](https://github.com/influxdata/influxdb/pull/25432), if you'd like to make any further suggestions please do it in that one 

https://github.com/influxdata/influxdb/blob/a8ce071a55e215688240d13a7a9c9772512ddfa7/influxdb3_telemetry/src/stats.rs#L105-L114